src/pages/workspaces/index.tsx

```diff
@@ -47,7 +48,14 @@ const Workspaces = () => {
             </FcBar>
             <MiniBdList>
               {drelloBoardsList.map((drelloBoardsItem) => (
-                <li key={drelloBoardsItem.id}>{drelloBoardsItem.title}</li>
+                <Link
+                  key={drelloBoardsItem.id}
+                  href={`/boards/${drelloBoardsItem.id}`}
```

Comment:
Want to use constant instead

Reply:
I made the change. Thanks for the reminder!